### DISCUSSION TITLE
Is Elastalert2 affected by new log4j vulnerability CVE-2021-44228 ?

### DISCUSSION THREAD

**Discussion by rschirin** [jertel/elastalert2#611] [Q&A] [answered]
Hey there,
following the subject, is Elastalert2 affected?

**Comment by jertel** [comment, marked as answer]
No, ElastAlert 2 does not use log4j, nor does it use Java. However, see Elastic's response to their products' vulnerability and mitigation requirements: https://discuss.elastic.co/t/apache-log4j2-remote-code-execution-rce-vulnerability-cve-2021-44228-esa-2021-31/291476
